roles/openshift_health_checker/test/fluentd_config_test.py

```diff
@@ -0,0 +1,325 @@
+import pytest
+
+from openshift_checks.fluentd_config import FluentdConfig, OpenShiftCheckException
+
+
+def canned_fluentd_config(exec_oc=None):
+    """Create a Fluentd check object with canned exec_oc method"""
+    check = FluentdConfig("dummy")  # fails if a module is actually invoked
+    if exec_oc:
+        check._exec_oc = exec_oc
+    return check
+
+
+def canned_fluentd_pod(containers=[]):
```

Comment:
You have a flake8 error here. Not a problem in this case, but sometimes mutable containers in arguments cause hairy problems...
Seems that we never call the function without an argument, so why don't simply make it required? (s/=[]//)